fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -811,6 +815,12 @@ public boolean getUseScanBlockCache() {
     @VarAttr(name = BIG_QUERY_LOG_SCAN_ROWS_THRESHOLD)
     private long bigQueryLogScanRowsThreshold = 1000000000L;
 
+    @VariableMgr.VarAttr(name = ICEBERG_VERSION_AS_OF)
+    private long icebergVersionAsOf = -1;
```

Comment:
starrocks has support as of syntax, I think you can get the version from there, not by the session variables.

Reply:
ok, I will update the code.